post-morterms/2018-03-30-sierra-api.md

```diff
@@ -0,0 +1,32 @@
+# No Bib/Item Platform Updates For 3 Days
+
+1. Incident Date
+   - March 30, 2018
+2. Incident Summary
+   - On March 27, 2018 at ~2:55pm, bib/item updates from Sierra stopped propagating to our Platform.
+   - It was discovered on March 30, 2018 at ~11:00am (almost 3 days later) that the primary Sierra API key used by our Platform services was accidently deleted.
+3. Author(s)
+   - Kevin Friedman
```

Comment:
Was this a one-person post-mortem?

Reply:
Might be worth having a note about how to conduct the post-mortem, not just the document that should arise.

Reply:
Ugh, unfortunately this was a one-person post-mortem. This is not ideal but since I was the one that primarily triaged, I decided to write this out. 

We do actually have a standard about writing post-mortems:
https://github.com/NYPL/engineering-general/blob/master/standards/postmortems.md

... of which I did not follow:
>SHOULD be done as a collaboration between engineers, managers, and parties affected.

Anyway, feel free to open an PR to the post-mortem standard if you have any ideas for improvements!

Reply:
No problem! It's great that you did it. And not every post-mortem needs a lot of people. Thanks for doing it!

Reply:
> Post-mortems are _blameless_. 

So is the practice of post-mortems :-)